jsk_pcl_ros/src/extract_indices_nodelet.cpp

```diff
@@ -82,7 +82,7 @@ namespace jsk_pcl_ros
     sub_indices_.unsubscribe();
   }
 
-  void ExtractIndices::convert(
+  bool ExtractIndices::convert(
```

Comment:
dont change the `void`.

Reply:
This change is awful since it bothers other dependencies sorry.

Reply:
yes, it must be `void`